mlflow/store/db_migrations/versions/33f7226d03b2_fix_killed_status_constraint.py

```diff
@@ -61,7 +61,6 @@ def upgrade():
         # database-backend-dependent enum (see https://docs.sqlalchemy.org/en/13/core/
         # type_basics.html#sqlalchemy.types.Enum.params.native_enum)
         batch_op.alter_column("status",
-                              existing_type=String,
```

Comment:
As per alembic docs, this seems fine to do:

> existing_type – Optional; a TypeEngine type object to specify the previous type. This is required for all MySQL column alter operations that don’t otherwise specify a new type, as well as for when nullability is being changed on a SQL Server column. It is also used if the type is a so-called SQLlchemy “schema” type which may define a constraint (i.e. Boolean, Enum), so that the constraint can be dropped.